crates/uv/src/commands/pip_compile.rs

```diff
@@ -393,7 +404,13 @@ pub(crate) async fn pip_compile(
 }
 
 /// Format the `uv` command used to generate the output file.
-fn cmd(include_index_url: bool, include_find_links: bool) -> String {
+#[allow(clippy::fn_params_excessive_bools)]
```

Comment:
I'm new to Rust and I didn't come up with a good idea to avoid the clippy warning. I'd appreciate it if you suggest a better way.

Reply:
I think what you have here is ok!Tthough I'm gonna remove the arguments because I think we should just always omit these.

Reply:
Okay! Thank you for your comment.